packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -82,6 +87,13 @@ export function ComboboxContent(props: ComboboxContentProps): JSX.Element {
         setSearchValue={setSearchValue}
       />
 
+      {multiselect && (
+        <ComboboxHeader
+          subjectNoun={props.subjectNoun}
+          selectedCount={selectedCount}
```

Comment:
nice I like this approach where we figure out the logic of the count out here, and `ComboboxHeader` gets to be very naive and only cares about a number

I'm not sure when exactly we'd reuse it, but if we did it makes this component not coupled to our Combobox implementation